.golangci.yml

```diff
@@ -1,6 +1,15 @@
 run:
-  tests: false
+  tests: true
   timeout: 5m
+  allow-parallel-runners: true
+  skip-files:  
+    - x/gamm/keeper/grpc_query_test.go
+    - x/gamm/client/cli/query_test.go
+    - x/gamm/client/cli/cli_test.go
+    - x/gamm/keeper/swap_test.go
```

Comment:
Those tests are skipped because the code generation currently makes a whole lot of deprecated code. In order to fix that, we would just need to update the query generator I think.

Reply:
By the way, looking at this list, I'm not 100% sure about the swap test file. I can have another look at it tomorrow. Try to figure out why that one is there.

Reply:
have this reduced to just the generated queries.